python/mage/link_prediction/training_visualizer.py

```diff
@@ -0,0 +1,51 @@
+from typing import List, Dict
+import matplotlib.pyplot as plt
+
+
+def visualize(training_results: List[Dict[str, float]], test_results: List[Dict[str, float]]) -> None:
+    """Visualize training results obtained during ML training. It assumes that training and test results are of same size=validated after same number of epochs.
+    Goal is to create drawing function that draws at most 4 subplots at the same figure and is dynamically adapting to the number of metrics sent.
+
+    Args:
+        training_results (List[Dict[str, float]]): Results obtained on training set. Contains various metrics for training.
+        test_results (List[Dict[str, float]]): Results obtained on test set.
+    """
+    tr_metrics, test_metrics = dict(), dict()
+    epochs = []
+
+    for epoch in range(len(training_results)):
+        epochs.append(epoch)
+        epoch_training_result = training_results[epoch]
+        epoch_test_result = test_results[epoch]
```

Comment:
This file is left for user to create. Keep it for yourself but put it in .gitignore. Similar should happen to model.pt